src/Nox.Generator/Presentation/Api/OData/EntityControllerGenerator.cs

```diff
@@ -251,7 +251,7 @@ private static void GeneratePut(Entity entity, CodeBuilder code, NoxSolution sol
     private static void GeneratePatch(Entity entity, string entityName, string pluralName, CodeBuilder code, NoxSolution solution)
     {
         // Method Patch
-        code.AppendLine($"public virtual async Task<ActionResult<{entity.Name}Dto>> Patch({PrimaryKeysFromRoute(entity, solution)}, [FromBody] Delta<{entityName}UpdateDto> {entity.Name.ToLowerFirstChar()})");
+        code.AppendLine($"public virtual async Task<ActionResult<{entity.Name}Dto>> Patch({PrimaryKeysFromRoute(entity, solution)}, [FromBody] Delta<{entityName}Dto> {entity.Name.ToLowerFirstChar()})");
```

Comment:
@AndreyDegtyarev see this change that will make deserialization possible

Reply:
Thanks Anna, I will try